src/objective-c/ProtoRPC/ProtoService.m

```diff
@@ -65,6 +65,8 @@ - (instancetype)initWithHost:(NSString *)host
   return self;
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wincompatible-pointer-types"
```

Comment:
```
method:methodName
```
triggers this warning. `methodName` has type 'ProtoMethod *', but 'GRPCProtoMethod *' is expected here. Looks like it's caused by the deprecated API.